inc/notificationeventabstract.class.php

```diff
@@ -62,11 +62,10 @@ static public function raise(
       global $CFG_GLPI;
       if ($CFG_GLPI['notifications_' . $options['mode']]) {
          $entity = $notificationtarget->getEntity();
+         $processed = [];
          if (isset($options['processed'])) {
-            $processed = &$options['processed'];
+            $processed = $options['processed'];
```

Comment:
No need for `&` if `$options['processed']` is unset.

Reply:
Not agree with this change. Array was sent to raiseEvent as empty one.

Log with your change  =>  both notifications are sent

==> mail.log <==
2019-03-29 09:50:33 [8@nelly]
Un courriel pour glpi@ll-experts.com a été ajouté à la file d'attente : [[NELLY] #0000327] new groupe nelly


==> php-errors.log <==
[2019-03-29 09:50:33] glpiphplog.DEBUG: NotificationEvent::raiseEvent() in /home/yllen/git94/inc/notificationevent.class.php line 165
processed johann Array
  (
      [mailing] => Array
          (
          )
  
  )
   {"user":"8@nelly"} 

==> mail.log <==
2019-03-29 09:50:33 [8@nelly]
Un courriel pour glpi@ll-experts.com a été ajouté à la file d'attente : [[NELLY] #0000327] Nouveau groupe Root

==> php-errors.log <==
[2019-03-29 09:50:33] glpiphplog.DEBUG: NotificationEvent::raiseEvent() in /home/yllen/git94/inc/notificationevent.class.php line 165
processed johann Array
  (
      [mailing] => Array
          (
          )
  
  )
   {"user":"8@nelly","mem_usage":"0.152\", 10.59Mio)"} 


Log with my commit => one notification of sub entity is sent

==> mail.log <==
2019-03-29 09:53:15 [8@nelly]
Un courriel pour glpi@ll-experts.com a été ajouté à la file d'attente : [[NELLY] #0000327] new groupe nelly

==> php-errors.log <==
[2019-03-29 09:53:15] glpiphplog.DEBUG: NotificationEvent::raiseEvent() in /home/yllen/git94/inc/notificationevent.class.php line 165
processed yllen Array
  (
      [mailing] => Array
          (
              [fr_FR] => Array
                  (
                      [glpi@ll-experts.com] => Array
                          (
                              [language] => fr_FR
                              [additionnaloption] => Array
                                  (
                                      [show_private] => 1
                                      [usertype] => 1
                                  )
  
                              [username] => MOED nelly
                              [users_id] => 8
                              [email] => glpi@ll-experts.com
                          )
  
                  )
  
          )
  
  )
   {"user":"8@nelly"} 
[2019-03-29 09:53:15] glpiphplog.DEBUG: NotificationEvent::raiseEvent() in /home/yllen/git94/inc/notificationevent.class.php line 165
processed yllen Array
  (
      [mailing] => Array
          (
              [fr_FR] => Array
                  (
                      [glpi@ll-experts.com] => Array
                          (
                              [language] => fr_FR
                              [additionnaloption] => Array
                                  (
                                      [show_private] => 1
                                      [usertype] => 1
                                  )
  
                              [username] => MOED nelly
                              [users_id] => 8
                              [email] => glpi@ll-experts.com
                          )
  
                  )
  
          )
  
  )
   {"user":"8@nelly","mem_usage":"0.019\", 10.48Mio)"}